src/Kinepolis/KinepolisService.php

```diff
@@ -58,6 +63,34 @@ public function import(): void
             $this->logger->error('Problem with Kinepolis Service Authentication: ' . $exception->getMessage());
             return;
         }
+
+        try {
+            $theaters = $this->client->getTheaters($token);
+        } catch (\Exception $exception) {
```

Comment:
Is there a more specific exception we could listen to instead of the generic Exception?
I would also just import the namespace.